mesheryctl/internal/cli/root/filter/list.go

```diff
@@ -54,7 +55,21 @@ mesheryctl exp filter list
 		if err != nil {
 			return err
 		}
-
+		prov := models.RemoteProvider{}
+		tokenPath := utils.AuthConfigFile
+		file, err := os.ReadFile(tokenPath)
+		if err != nil {
+			return err
+		}
+		jwtToken := map[string]string{}
+		errCheck := json.Unmarshal(file, &jwtToken)
+		if errCheck != nil {
+			return ErrUnmarshal(errCheck)
+		}
+		_, invalid := prov.VerifyToken(jwtToken["token"])
+		if invalid != nil {
+			return ErrInvalidAuthToken()
+		}
 		res, err := client.Do(req)
```

Comment:
@suhail34 This logic is redundant as @leecalcote has pointed out. Use this function instead - 

https://github.com/meshery/meshery/blob/5c11aa7e6f1ff2c1cd3cb3f9990abfb05153a1f0/mesheryctl/pkg/utils/auth.go#L65-L94